src/twisted/conch/checkers.py

```diff
@@ -580,8 +584,7 @@ def _verifyKey(self, pubKey, credentials):
         try:
             if pubKey.verify(credentials.signature, credentials.sigData):
                 return credentials.username
-        except:  # Any error should be treated as a failed login
-            log.err()
+        except Exception:  # Any error should be treated as a failed login
             raise UnauthorizedLogin('Error while verifying key')
```

Comment:
Exception chaining here?

```suggestion
        except Exception as e:  # Any error should be treated as a failed login
            raise UnauthorizedLogin('Error while verifying key') from e
```